itests/org.openhab.core.config.discovery.usbserial.tests/itest.bndrun

```diff
@@ -5,43 +5,51 @@ Fragment-Host: org.openhab.core.config.discovery.usbserial
 
 -runrequires: bnd.identity;id='org.openhab.core.config.discovery.usbserial.tests'
 
+# We would like to use a mock instead of the "PollingUsbSerialScanner" in tests
+# So make sure 'org.openhab.core.config.discovery.usbserial.linuxsysfs' is not in runbundles
```

Comment:
I tried blacklisting the bundle here but then resolving bundles always fails. So now this bundle needs to be manually removed after resolving bundles. If you know a better way please comment. :-)

Reply:
Hm, I am not aware of such a configuration. May @maggu2810 has an idea. Otherwise we can merge this. I do not expect much upcoming changes to in these bundles / itests.